docs/oauth2.md

```diff
@@ -342,11 +335,165 @@ a write permission on all exchanges starting with `x-prod-`, and any routing key
 ```
 
 
+### Scope and Tags {#scope-and-tags}
+
+Users in RabbitMQ can have [tags associated with them](./access-control#user-tags).
+Tags are used to [control access to the management plugin](./management#permissions).
+
+In the OAuth context, tags can be added as part of the scope, using a format like `&lt;resource_server_id>.tag:&lt;tag>`. For example, if `resource_server_id` is "my_rabbit", a scope to grant access to the management plugin with
+the `monitoring` tag will be `my_rabbit.tag:monitoring`.
+
+## Basic usage {#basic-usage}
+
+### Configure OAuth 2.0 provider's issuer {#configure-issuer}
+
+Before RabbitMQ 3.13, users had to set the JWKS endpoint (i.e. `auth_oauth2.jwks_url` setting) or statically [configure the signing keys](#configure-signing-keys). Now, users can instead configure the OpenID Provider's issuer URL and from this URL RabbitMQ downloads the OpenID Provider configuration which includes the JWKS endpoint in addition to other endpoints which will be useful in other contexts.
+
+Typically, this same `issuer` URL has been the same URL configured in the management plugin (`management.oauth_provider_url`). From now on, it is only necessary to configure a single URL, specified by the `auth_oauth2.issuer` setting.
+
+Sample configuration using issuer:
+```ini
+auth_oauth2.resource_server_id = my_rabbit_server
+auth_oauth2.issuer = https://my-idp-provider/somerealm
+```
+
+or
+
+```erlang
+[
+  {rabbitmq_auth_backend_oauth2, [
+    {resource_server_id, <<"my_rabbit_server">>},
+    {issuer, <<"https://my-idp-provider/somerealm">>}    
+  ]},
+].
+```
+
+Sample configuration using jwks_url:
+```ini
+auth_oauth2.resource_server_id = my_rabbit_server
+auth_oauth2.jwks_url = "https://my-jwt-issuer/jwks.json
+```
+
+or
+
+```erlang
+[
+  {rabbitmq_auth_backend_oauth2, [
+    {resource_server_id, <<"my_rabbit_server">>},
+    {key_config, [
+      {jwks_url, <<"https://my-jwt-issuer/jwks.json">>}
+    ]}
+  ]},
+].
+```
+
+**Note**: If you have both endpoints configured, RabbitMQ will use `jwks_url` given that it does not have to discover it via the `issuer` url.
+
+**Note about TLS settings for the `jwks_url` or the `issuer` url**: TLS setting such as the `cacertfile` are configured as follows regardless which url we are using:
+
+```ini
+...
+auth_oauth2.https.cacertfile = /opts/certs/cacert.pem
+...
+```
+
+or
+
+```erlang
+[
+  {rabbitmq_auth_backend_oauth2, [
+    ...
+    {key_config, [
+      {cacertfile, <<"/opts/certs/cacert.pem">>}
+    ]}
+  ]},
+].
+```
+
+**VERY IMPORTANT**: Since RabbitMQ 3.13, if `auth_oauth2.https.peer_verification` setting is not set, RabbitMQ sets it to `verify_peer` as long as there are trusted certificates installed in the OS or the user configured `auth_oauth2.https.cacertfile`. Because it is very common to have trusted certificates installed in the OS, effectively, RabbitMQ will always enforce certificate validation of the OAuth Provider, unless you
```

Comment:
try not to start a sentence with "Because", I don't think you even need that word, start with "It is..."